### PR TITLE
Bluetooth: Audio: Miscellaneous modification

### PR DESCRIPTION
**Pull request by LingaoM**
Miscellaneous modification.

### CODE REVIEW

File: subsys/bluetooth/audio/Kconfig [hunk 1]
```diff
@@ -17,31 +17,42 @@ menuconfig BT_AUDIO
 
 if BT_AUDIO
 
-if BT_CONN
-
 config BT_AUDIO_UNICAST
-	bool "Bluetooth Unicast Audio Support"
+	bool # Hidden
```
Comment (Thalley) [inline subsys/bluetooth/audio/Kconfig:21]: Personally I'm a fan of using `# Hidden` for Kconfigs, but I think that someone just pruned a lot of these upstream as it apparently is implied.

**Review by Thalley — APPROVED** [review]
Had a small comment, otherwise these changes look great. Thanks!

File: subsys/bluetooth/audio/Kconfig (hunk 1 above)
Comment (LingaoM) [reply subsys/bluetooth/audio/Kconfig:21]: Do I need to change it?

Comment (Thalley) [reply subsys/bluetooth/audio/Kconfig:21]: Not for my sake; it's up to you, but we may get requested to remove it when we move this in the `main` branch.

**Comment by LingaoM** [comment]
@Thalley Build CI failed, but look like irrelevant with this pr (https://buildkite.com/zephyr/zephyr/builds/41341#9dbce099-e9f4-47e8-a445-fdd9b99b6c75)

**Comment by Thalley** [comment]
> @Thalley Build CI failed, but look like irrelevant with this pr (https://buildkite.com/zephyr/zephyr/builds/41341#9dbce099-e9f4-47e8-a445-fdd9b99b6c75)

Yeah, I can see that. I restarted the tests for you (didn't know if you had permissions to do that). Unfortunately we don't yet have any tests for BAPS in CI (except for building the audio_shell project). Let's see if they pass this time :)

**Pull request merged**